setup.py

```diff
@@ -306,7 +310,11 @@ def run(self):
 
 ext_modules = [
     custom_calls_extension,
-    CMakeExtension("catalyst.utils.wrapper", sourcedir=frontend_dir),
+    CMakeExtension(
+        "catalyst.utils.wrapper",
+        sourcedir=frontend_dir,
+        define_macros=Py_LIMITED_API_macros,
+    ),
```

Comment:
Just to confirm, to your knowledge @joeycarter the only module that wasn't already being built with the limited API was the custom calls then?

Reply:
Removed

Reply:
@dime10, yes, as far as I know we do not need to explicitly opt in to the limited API anywhere else. In fact, it might not even be strictly necessary for custom_calls, since it doesn't actually use the Python C API at the moment, but it makes our intention explicit should this change in the future.